lib/brook.ex

```diff
@@ -225,4 +225,32 @@ defmodule Brook do
       {:error, reason} -> raise reason
     end
   end
+
+  @spec serialize(term) :: {:ok, String.t()} | {:error, term}
+  def serialize(data) do
+    case Application.get_env(:brook, :serializer, :brook) do
```

Comment:
wouldn't it be better to set the default to json_serde if you prefer that as your go-forward?

Reply:
maybe, I was thinking of smart columbus and figured it they forget to set it to brook, it wouldn't cause any problems for them, but I could persuaded to change the default

Reply:
i'm thinking from the perspective of our other libraries where we've shied away from this in general vs. smart columbus where they already have very crowded app env configs so it doesn't add much burden to them to include another setting.

Reply:
I agree with that point @jeffgrunewald but I think that's the wrong reason to keep this default. To me, it makes a lot of sense that the serializer with brook in its name is the default. Anything other than that, and it goes against what a non-contributor would expect.